Plugins/ExaTrkX/src/OnnxEdgeClassifier.cpp

```diff
@@ -80,12 +76,12 @@ OnnxEdgeClassifier::OnnxEdgeClassifier(const Config &cfg,
   sessionOptions.SetGraphOptimizationLevel(
       GraphOptimizationLevel::ORT_ENABLE_EXTENDED);
 
-  if (torch::cuda::is_available()) {
-    ACTS_INFO("Try to add ONNX execution provider for CUDA");
-    OrtCUDAProviderOptions cuda_options;
-    cuda_options.device_id = 0;
-    sessionOptions.AppendExecutionProvider_CUDA(cuda_options);
-  }
+#ifndef ACTS_EXATRKX_CPUONLY
```

Comment:
this is not purely compile time dependent?

I think usually we try to avoid `#ifndef` in the Core but might be relaxed for plugins

Reply:
In some cases conditionally adding complication units can help but not always.